README.md

```diff
@@ -29,7 +57,7 @@ source env_setup_acc.sh
 ./install_dep_packages_acc.sh
 ```
 
-Clone strongSwan and switch to version 5.9.3 (Any version can be used but verification has been completed with v5.9.3 on fedora33)
+Clone strongSwan and switch to version 5.9.3 (any version can be used but verification has been completed with v5.9.3 on Fedora 33)
```

Comment:
```suggestion
Clone strongSwan and switch to version 5.9.3 (any version can be used, but verification has been completed with v5.9.3 on Fedora 33)
```
Quibble: one doesn't switch to a _version_ in git, one switches to a branch or tag. Just sayin'.